addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1419,6 +1419,20 @@ var SnippetsMenu = Widget.extend({
         $hook.replaceWith($dropzone);
         return $dropzone;
     },
+    /**
+     * Installs a module
+     *
+     * @private
+     * @param {number} moduleID: the module to install
+     * @returns {Promise}
+     */
+    _installModule: function (moduleID) {
```

Comment:
In fact, that method is not useful as only called one time, do the RPC directly in the handler :+1: